.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```

Comment:
I see this `.idea` directory. Does this directory need to be tracked in source control? Do I need to be using a JetBrains IDE to work with this project?

Reply:
I would do a little research on your own to see why you would/wouldn't want to include the `.idea` directory in source control. If you decide you want to exclude it, then yes I would remove the directory and files from git tracking and include it in the `.gitignore`.

Reply:
From what I looked into, you can keep the .idea tracked if you want to, but some files within it probably shouldn't be included as they are specifically for your IntelliJ configuration (like the workspace.xml, tasks.xml) which my  .gitignore in my .idea folder takes care of. I also often saw people say they just avoided it all together, but that doesn't seem quite right to me.

Reply:
Hmmm, now the more I look into it the more I might say to not include it all together, or at least most of it. Through a couple of links I found this link that generates a git ignore file automatically depending on the IDE you put in: https://www.toptal.com/developers/gitignore which included a bit more than I thought originally, could be a useful resource 🤔

Reply:
Sounds good to me, that is what I was leaning towards as well. Thank you for the explanation too, that makes a lot of sense! I am thinking of making this change its own PR (and put it as a ticket to keep track of it) since it is more of an organizational change (maybe putting it with the mock organization change) and i feel like it doesn't really belong in this PR. What are your thoughts?

Reply:
Going to defer this to be included in a separate PR as discussed above, Potential ticket name (for tracking purposes) -> untrack-idea-directory ticket, thank you all for the help!